utils/api_requests.py

```
@@ -62,81 +65,106 @@
     return auction_info["auctions"]
 
 
-def get_update_timers(region, simple_snipe=False):
-    # get from api every time
+def get_petnames(access_token):
+    headers = {"Authorization": f"Bearer {access_token}"}
+    pet_info = requests.get(
+        f"https://us.api.blizzard.com/data/wow/pet/index?namespace=static-us&locale=en_US",
```

Ruff

f-string without any placeholders

Remove extraneous `f` prefix

(F541)